config/crd/bases/dynatrace.com_edgeconnects.yaml

```diff
@@ -339,7 +338,7 @@ spec:
                       Requests describes the minimum amount of compute resources required.
                       If Requests is omitted for a container, it defaults to Limits if that is explicitly specified,
                       otherwise to an implementation-defined value. Requests cannot exceed Limits.
-                      More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+                      More info: https://kubernetes.
```

Comment:
Are these links cut intentionally?

Reply:
it's due to this change https://github.com/Dynatrace/dynatrace-operator/pull/4084/files#diff-6118328086c863678a3dcf8134f32c75dc0558aed56214e0becfa6062161b9deR1
we cut entire description :(

Reply:
Sad.
I would remove then completely all these lines
```
More info: https://kubernetes.
```
just IMO

Reply:
it's done automatically, i can't edit this file.